addons/rating/migrations/11.0.1.0/pre-migration.py

```diff
@@ -14,3 +14,18 @@
 @openupgrade.migrate()
 def migrate(env, version):
     openupgrade.copy_columns(env.cr, COLUMN_COPIES)
+
+    # Precreate and populate mail_message's new stored field rating_value
+    env.cr.execute(
```

Comment:
Maybe you can use the openupgradelib function that saves you also to handle the default value and thus the latest query:
```python
openupgrade.add_fields(env, [("rating_value", "mail.message", "mail_message", "float", False, "rating", 0)])
```
but I don't want to be picky, so if you don't want to change this, we can merge it as is.

Reply:
Yes, alright.